lib/pages/people/view/people_page.dart

```diff
@@ -55,6 +82,63 @@ class _PeoplePageState extends State<PeoplePage> {
     );
   }
 
+  List<Person> get filteredPeople => peopleData
+      .where((person) => filter
+          .split(' ')
+          .where((element) => element != '')
+          .fold(
+              true,
+              (previousValue, filter) =>
+                  previousValue && person.name.toLowerCase().contains(filter)))
+      .toList();
+}
+
+class PeopleList extends StatefulWidget {
+  const PeopleList({this.people, this.filter});
+
+  final List<Person> people;
+  final String filter;
+
+  @override
+  _PeopleListState createState() => _PeopleListState();
+}
+
+class _PeopleListState extends State<PeopleList> {
+  @override
+  Widget build(BuildContext context) {
+    final List<String> filteredWords =
```

Comment:
You need to make these lowercase, otherwise if I search someone's first name with uppercase it won't find it).